kolibri/core/device/utils.py

```diff
@@ -22,6 +22,8 @@ def get_device_setting(setting, default=no_default_value):
 
     try:
         device_settings = DeviceSettings.objects.get()
+        if device_settings is None:
+            raise DeviceSettings.DoesNotExist
```

Comment:
Can `objects.get` ever return `None`?

Isn't it the `getattr` below that returns `None`?

Reply:
The `get` method is overwritten to implement the caching, so the assumption was the cache was returning `None` at some point.